inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java

```diff
@@ -450,7 +426,23 @@ protected GraphPattern rootsPattern(KnowledgeBase aKb)
             }            
         }
     }
-    
+
+    private static void selectValue(KnowledgeBase aKB, Iri aContext, List<GraphPattern> classPatterns) {
```

Comment:
I think you should refactor these two methods into one. Maybe you could have a method like `private void createPattern(KnowledgeBase aKB, RdfSubject subject, RdfObject object, List<GraphPattern> patterns)`. When called, subject and object can be defined as needed.